python/lsst/sconsUtils/dependencies.py

```diff
@@ -334,6 +348,72 @@ def configure(self, conf, packages, check=False, build=True):
                     return False
         return True
 
+    @staticmethod
+    def _get_config_var(name):
+        """The relevant Python is not guaranteed to be the Python
+        that we are using to run SCons so we must shell out to the
+        PATH python."""
+        pycmd = 'import distutils.sysconfig as s; print(s.get_config_var("{}"))'.format(name)
+        result = subprocess.check_output(["python", "-c", pycmd]).decode().strip()
+        # Be consistent with native interface
+        if result == "None":
+            result = None
+        return result
+
+    def configurePython(self, conf, packages, check=False, build=True):
+        state.log.info("Configuring package '%s'." % self.name)
+        python3rdinclude = self._get_config_var("CONFINCLUDEPY")
+        conf.env.AppendUnique(XCPPPATH=python3rdinclude)
+        conf.env.AppendUnique(XCPPPATH=python3rdinclude + "/..")
+        conf.env.AppendUnique(XCPPPATH=python3rdinclude + "/../eigen3")
+        usedC = state.env['CXX']
+        coutput = subprocess.run('which ' + usedC, shell=True, stdout=subprocess.PIPE)
```

Comment:
We have found that `which` is not always installed so this might trigger problems on some minimalist installations. We switched all our code over to `command -v` to avoid this problem.  It's almost simpler and safer to scan PATH yourself.